src/AppIcon.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<svg width="24"
+		height="24"
+		viewBox="0 0 24 24">
+		<image x="0"
+			y="0"
+			width="24"
+			height="24"
+			preserveAspectRatio="xMinYMin meet"
```

Comment:
```suggestion
```
When I remove this then the core icon is centered for me as well (and all other icons still look correct to me).